### PR TITLE
updated bit version

### PR DESCRIPTION
**Pull request by amitgilad3**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/bit.rb [hunk 1]
```diff
@@ -2,15 +2,8 @@
...
-  url "http://assets.bitsrc.io/release/0.1.0/bit_0.1.0_brew.tar.gz"
-  sha256 "c33c00a8334da4659b263472d013eaafbc5f0c16249e755f7624ac396d673270"
-
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "521284d178e8899881ad7a1461ac3a4387661b489e2a9e79675ff967883edfbc" => :sierra
-    sha256 "06a6e114c0ce663a7bda3cc422b3c47c1c16eca598305b191e0d508baf6470d8" => :el_capitan
-    sha256 "86fc112094a34ee836bfe23abb3920424a50836c8c4f1a07384d558c13f47be7" => :yosemite
-  end
+  url "http://104.154.76.155:8081/artifactory/bit-brew/stable/bit/0.1.7/bit-0.1.7-brew.tar.gz"
```
Comment (woodruffw) [inline Formula/bit.rb]: This probably isn't an acceptable URL. Is there an official release on the project's domain (i.e., `bitsrc.io`)?

File: Formula/bit.rb [hunk 2]
```diff
@@ -2,15 +2,8 @@
 class Bit < Formula
   desc "Distributed Code Component Manager"
   homepage "https://www.bitsrc.io"
-  url "http://assets.bitsrc.io/release/0.1.0/bit_0.1.0_brew.tar.gz"
-  sha256 "c33c00a8334da4659b263472d013eaafbc5f0c16249e755f7624ac396d673270"
-
-  bottle do
```
Comment (woodruffw) [inline Formula/bit.rb]: You don't need to remove this; `test-bot` will automatically rewrite these lines.

File: Formula/bit.rb (hunk 1 above)
Comment (nijikon) [inline Formula/bit.rb]: Yup, this will not do, sorry.

**Review by nijikon — CHANGES_REQUESTED** [review]

File: Formula/bit.rb (hunk 1 above)
Comment (amitgilad3) [inline Formula/bit.rb]: sorry , will fix and push new file

**Pull request closed without merging**